### DISCUSSION TITLE
Error in using MultiAppCopyTransfer

### DISCUSSION THREAD

**Discussion by avtarsinghh1991** [idaholab/moose#20964] [Q&A Modules: General] [answered]
Hello MOOSE Experts,

I am trying to transfer the data from sub_app (DIM_sub_particle) to main_app (test_2d) using following

```
[MultiApps]
  [full_solve]
    type = FullSolveMultiApp
    execute_on = initial
    positions = '0 0 0'
    input_files = DIM_sub_particle.i
    clone_master_mesh = true
  []
[]

[Transfers]
[./from_eta_gb]
    type = MultiAppCopyTransfer
    source_variable = eta_gb
    variable = eta_gb
    from_multi_app = 'DIM_sub_particle'
  [../]
    [./from_eta_pmi]
    type = MultiAppCopyTransfer
    source_variable = eta_pmi
    variable = eta_pmi
    from_multi_app = 'DIM_sub_particle'
  [../]
[]
```

However, I am getting the following error
```
application called MPI_Abort(MPI_COMM_WORLD, 1) - process 1

*** ERROR ***
Unable to locate object: DIM_sub_particle.
```
DIM_sub_particle file is there in the same folder. I was using it earlier with older syntax but new syntax its not working.

Please let me know if I am doing it right and suggest some solutions.

Best
Avtar

**Comment by GiudGiud** [comment, marked as answer]
Hello

The name of the sub app is `full_solve` not DIM_sub_particle

You just need to change that in to/from_multiapp in the transfers. 

Guillaume

**Reply by avtarsinghh1991** [reply]
Hi @GiudGiud 

Thank you very much. Its working now.

Best
Avtar
